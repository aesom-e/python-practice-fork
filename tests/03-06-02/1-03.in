Hello, World. The letters in this should look scrambled
-2
1
-4