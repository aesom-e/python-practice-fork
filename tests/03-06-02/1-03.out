Ffhjp, Smshb. Udc maruapt el udgt ofpqje hmpg qdnynxjfz